src/pages/Home/Home.tsx

```diff
@@ -73,7 +75,10 @@ const Home = () => {
    */
   useEffect(() => {
     if (store.dataset.datasetsLoaded) return;
-    fetch(`${API_URL}/datasets`)
+    fetch(`${API_URL}/datasets`, {
+      method: 'GET',
+      headers: { Authorization: `Bearer ${auth.user?.access_token}` },
```

Comment:
Javascript string templates are weird, this will evaluate to `Bearer undefined` in the event that authentication isn't defined. You probably only want to set this header in the condition that `auth.user?.access_token` is defined. i.e.:

```typescript
// ...
const fetchParams = {method: 'GET'};
if (auth.user?.access_token) {
  fetchParams["headers"] = {Authorization: `Bearer ${auth.user!.access_token}`}
}
fetch (`${API_URL}/datasets`, fetchParams)
// ...
```

Reply:
Alright, I think I addressed this.

Reply:
I was running into key issues for a minute (I think because method and then "headers" are different types maybe), but I decided to just construct the fetch params all at once.